controllers:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
  ur5_controllers:
    type: UR5/UR5CONTROLLER
 
 